index.js

```diff
@@ -141,7 +141,7 @@ module.exports = {
   },
 
   getParentAddonTree() {
-    return path.join(this.parentAddon.root, this.parentAddon.treePaths.addon);
+    return path.join(this.parentAddon.root, this.parentAddon.treePaths.src || this.parentAddon.treePaths.addon);
```

Comment:
This does in fact not work classic apps consuming MU addons and classic addons, as `src` is apparently always defined. We would need a way to detect, whether the current addon is classic or MU.